src/custom/pages/App/index.tsx

```diff
@@ -20,7 +20,7 @@ const PrivacyPolicy = lazy(() => import(/* webpackChunkName: "privacy_policy" */
 const CookiePolicy = lazy(() => import(/* webpackChunkName: "cookie_policy" */ 'pages/CookiePolicy'))
 const TermsAndConditions = lazy(() => import(/* webpackChunkName: "terms" */ 'pages/TermsAndConditions'))
 const About = lazy(() => import(/* webpackChunkName: "about" */ 'pages/About'))
-const Profile = lazy(() => import(/* webpackChunkName: "profile" */ 'pages/Profile'))
+const Account = lazy(() => import(/* webpackChunkName: "profile" */ '@src/custom/pages/Account'))
```

Comment:
```suggestion
const Account = lazy(() => import(/* webpackChunkName: "profile" */ 'pages/Account'))
```

Reply:
This is how it looks like in the last branch (`improve-header-12`)

> const Account = lazy(() => import(/* webpackChunkName: "profile" */ '@src/custom/pages/Account'))

Let me know if that should be changed to your suggestion:
> const Account = lazy(() => import(/* webpackChunkName: "profile" */ 'pages/Account'))